tools/CAT/src/pages/EditPage/index.tsx

```diff
@@ -29,6 +29,11 @@ import {
 import { fetchCategoriesData } from '@/app/features/categories/categoryThunks';
 import { showErrorNotif } from '@/providers/notify';
 
+enum TemplateType {
+	DEFAULT = 0,
+	WITH_TITLE = 1,
+}
+
```

Comment:
We already have this enumeration. Les`s use it

![image](https://github.com/DzeshnikSoft/college-chnu/assets/64914675/7956c6d5-7598-4982-b28e-fb00c0031a59)